SecretManagement.LastPass.psm1

```diff
@@ -216,6 +216,85 @@ function Get-VaultParams {
 
 }
 
+<#
+.SYNOPSIS
+Show LastPass Grid view secrets then show the selected secret.
+
+.DESCRIPTION
+Show LastPass Grid view secrets then show the selected secret. 
+
+.PARAMETER Vault
+Name of the vault used for the lookup
+
+.PARAMETER Filter
+Pre-filter secrets based on the specified keywords
+
+.PARAMETER KeepOpen
+If set, the secrets GridView will be automatically reloaded after a secret is shown.
+
+.PARAMETER PassThru
+If set, Secret will be returned as is without formatting.
+
+.EXAMPLE
+Show-LastPassConsoleGridView -Vault MyVault -KeepOpen 
+
+.NOTES
+This cmdlet can make use of the improved Out-ConsoleGridView cmdlet if using Powershell 6.2 or newer and  Microsoft.PowerShell.ConsoleGuiTools is installed.
+Otherwise, Out-GridView will be used.
+#>
+Function Show-LastPassConsoleGridView {
+    [CmdletBinding(DefaultParameterSetName='Default')]
+    Param(
+        [String]$Vault,
+        [String]$Filter = '*',
```

Comment:
You add star to the end of $Filter down below so I don't think you need to set this default for this parameter since:

```
"$filter" -eq ""
```
when `$Filter` is undefined

Reply:
https://github.com/TylerLeonhardt/SecretManagement.LastPass/pull/23/commits/a601d6098f5600db1249874a5564b8ed965c5716